spec/reek/cli/command/todo_list_command_spec.rb

```diff
@@ -37,6 +37,40 @@
         result = command.execute
         expect(result).to eq(Reek::CLI::Options::DEFAULT_SUCCESS_EXIT_CODE)
       end
+
+      it 'writes a todo file' do
+        command.execute
+        expected_yaml = { 'FeatureEnvy' => { 'exclude' => ['self'] } }.to_yaml
```

Comment:
The hash doesn't have the same structure as SmellWarning's attributes, so I'll add an explicit value for the context to the smell instead.